import-mailbox-to-gmail.py

```diff
@@ -159,6 +189,39 @@ def get_label_id_from_name(service, username, labels, labelname):
     raise
 
 
+# TODO: instead of passing mbox_label_name as param, can't we look it up from id?
```

Comment:
We can, but it would be less efficient (potentially much slower if you have many labels and many messages). I think we can drop this comment unless you feel strongly about it.